android/app/src/main/res/layout/activity_main.xml

```diff
@@ -8,18 +8,11 @@
     android:fitsSystemWindows="true"
     tools:openDrawer="start">
 
-    <android.support.design.widget.CoordinatorLayout
+    <FrameLayout
```

Comment:
Why to change CoordinateLayout to FrameLayout ?

CoordinateLayout is super powered FrameLayout. 
Which enables many features like dismiss snackbar using swiping